conf/defaults.yaml

```diff
@@ -39,7 +39,7 @@ storm.exhibitor.port: 8080
 storm.exhibitor.poll.uripath: "/exhibitor/v1/cluster/list"
 storm.cluster.mode: "distributed" # can be distributed or local
 storm.local.mode.zmq: false
-storm.thrift.transport: "org.apache.storm.security.auth.SimpleTransportPlugin"
+storm.thrift.transport: "org.apache.storm.security.auth.plain.PlainSaslTransportPlugin"
```

Comment:
Since there is not good place to document this. I am simply going without any chances to default on "storm.thrift.tranport" so we can pull this in 1.x-branch as well. I will create two separate pull requests - 
1. to change default and deprecate  `SimpleTransportPlugin`  and configuration on 1.x-branch.
2. To remove `SimpleTransportPlugin` and configuration in master.